modules/geth/README.md

```diff
@@ -0,0 +1,56 @@
+<!--
+title: "Go-ethereum monitoring with Netdata"
+description: "Monitor the health and performance of your go-ethereum Nodes (Geth) with zero configuration, per-second metric granularity, and interactive visualizations."
+custom_edit_url: https://github.com/netdata/go.d.plugin/edit/master/modules/geth/README.md
+sidebar_label: "Go-ethereum"
+-->
+
+# Geth Monitoring with Netdata
+
+
+Go Ethereum, written in Google’s Go language, is one of the three original implementations of the Ethereum protocol, alongside C++ and Python.
+
+Go-Ethereum, and subsequently Geth, are built and maintained by the Ethereum community. It’s open source which means anyone can contribute to Geth through its [Github](https://github.com/ethereum/go-ethereum).
+
+With Netdata,  you can effortlessly monitor your Geth node
+
+
+## Requirements
+
+Run `geth` with the flag `--metrics`. That will enable the metric server, with default port `6060` and path `/debug/metrics/prometheus`. 
+
+## Charts
+
+This is an initial number of metrics that we chose to collect and organize. It is **very easy** to add more charts based on the available metrics in the prometheus endpoint. Head over to [Contribute](#contribute) to learn how you can help to expand this collector. 
+
+- Chaindata: 
+  - total read/write for the session
+  - read/write per second
+- Transaction Pool
+  - Pending
+  - Queued
+- Peer-to-Peer
+  - bandwidth per second (ingress/egress)
+  - number of peers
+  - serves/dials calls per second
+- rpc calls
+  - successful/failed per second
+- reorgs
+  - Total number of executed reorgs
+  - Total number of added/removed blocks due to reorg
+- number of active goroutines
+- chainhead
+  - block, receipt and header. If block = header, then Geth node is fully synced. 
+
+## Contribute
```

Comment:
It is a common pattern to add [Configuration](https://github.com/netdata/go.d.plugin/tree/master/modules/wmi#configuration) and Troubleshooting[](https://github.com/netdata/go.d.plugin/tree/master/modules/wmi#troubleshooting) sections to to readme's.